.github/workflows/build.yaml

```diff
@@ -46,3 +51,11 @@ jobs:
           path: |
             target/failsafe-reports/
             target/surefire-reports/
+
+      - name: Attach Build Output
+        uses: actions/upload-artifact@v2
+        with:
+          name: target
+          path: |
+            target/
```

Comment:
@MikeEdgar  I'm a little conflicted if this is fine as it might be redundant in case of build failure situations. What do think?

Reply:
@biswassri you mean if it fails don't bother attaching the output? I don't disagree, were you thinking to add `if: !failure()` or similar to the step?

Reply:
Yes, so in case of failure it only attaches the failsafe-reports, like in the previous job `Archive result` and on success the above below attached.

Reply:
The `target` archive is only 5MB for this build. In my opinion it should be OK to attach it regardless of the outcome.

Reply:
Right makes sense! So, I'll leave it as is.